README.md

```diff
@@ -40,20 +44,28 @@ The baseline for this dataset is Molecule Attention Transformer, fine-tuned with
 The baseline for this dataset is Audio Spectrogram Transformer, more details can be found in [paper](https://arxiv.org/abs/2104.01778).
 
 ## Question 1
-Can pretrained language models transfer to different modalities?
+* Can pretrained language models transfer to different modalities?
+* Only input embedding layer, output readout layer and layer norm parameters are fine-tuned.
+* FPT archives comparable performance to the fully trained transformer.
```

Comment:
It's a conclusion, not a question. And where is the experimental justification for such conclusion?

Reply:
In the paper authors formulate such thesis and then they justify it by comparing baselines with FPT.

Reply:
As far as I understand our question: we do not compare fully trained transformers to FPT here. So this thesis (which I believe should be presented as a question) cannot be verified within our question.